index.rst

```diff
@@ -302,107 +305,156 @@ Is this case, we could have something like:
 Users will usually count on being able to easily enable a component with appropriate defaults first and then, what different configurations they have available to fine tune the behavior of the system.
 The use of *recommended* gives the impression that not everything that is shown is what is available (which is true in some cases), and also means users must look into the configuration repository to know what else is available.
 On the other hand *settings* really seems like a misnomer for *configuration*.
 
-The following is a renaming suggestion for discussion:
+The following list details the changes to the current implementation.
+Besides the renaming of numerous events and attributes, the most significant change is the removal of the labels associated with configuration files.
 
 #.  Rename ``settingsVersions`` to ``configurationsAvailable``.
 
     This topic presents **all** the available configurations that can be loaded by the CSC (see :ref:`the proposal <section-default-configuration>` to change the way CSC handles configuration).
+    As will be discussed in the following section, only the files that override the initial and site specific values will be displayed.
+
+    #.  Remove ``recommendedSettingsLabels`` and all notions of labels
+    #.  Add ``overrides``
+
+        - This will consist of a comma separated list of all configuration files in the configuration repo that can be loaded as overrides (discussed below)
+
+    #.  Rename ``recommendedSettingsVersion`` to ``versions``
 
-    #.  Rename ``recommendedSettingsLabels`` to ``labels``.
-    #.  Rename ``recommendedSettingsVersion`` to ``versions``.
-    #.  Rename ``settingsUrl`` to ``url``.
-    #.  Add ``mapping``.
-        For a configuration repository ``mapping`` will reflect the content of the ``_labels.yaml`` file.
-        For a configuration database ``mapping`` will explicitly show the mapping of ``label`` to ``label:version``.
+        - This will consist of the git hash associated with the commit of configuration repo that can be loaded as overrides (discussed below).
+
+    #.  Rename ``settingsUrl`` to ``url``
+    #.  Add ``schemaVersion`` which indicates the schema version in use (e.g. v3)
 
 #.  Rename ``settingsApplied`` to ``configurationApplied``
 
-    #.  Add ``label``.
-    #.  Add ``version``.
-    #.  Add ``mapping``.
-    #.  Add ``url``.
+    #.  Add ``configurations``
+        - This will consist of a comma separated list of between one and three file names (discussed below)
+
+    #.  Add ``version``
+    #.  Add ``url``
+    #.  Add ``schemaVersion``
     #.  Rename ``otherSettingsEvent`` -> ``otherInfo``.
 
     The event will publish the selected values once the CSC is configured.
 
-#.  In the ``start`` command, rename attribute ``settingsToApply`` to ``configuration``.
-
+#.  In the ``start`` command, rename attribute ``settingsToApply`` to ``configurationOverride``.
```

Comment:
I think we agreed with @r-owen that we want the word "override" in the parameter, but having only "override" was too ambiguous. 
https://lsstc.slack.com/archives/GFCFYPB8Q/p1618962157071900
All the other names use "configuration" so I'm hesitant to shorten only this instance. Seeing as this is now supposed to be rarely used am leaning more towards maintaining consistency and understanding rather than length...

Reply:
I personally prefer the verbosity... but if Tiago really thinks 7 characters will cause carpal-tunnel then I'm ok with changing everything to `config`